src/apps/support.ts

```diff
@@ -31,6 +31,7 @@ const appVersionsRequired = {
   Algorand: ">= 1.2.9",
   Polkadot: ">= 9.9140.0",
   Elrond: ">= 1.0.11",
+  Ethereum: ">= 1.9.17",
```

Comment:
Eth app 1.9.17 will now be mandatory for all eth tx because we don't have a flow that would allow us to determine 1) what device we're using 2) what the transaction is about, both at the same time and place.
Which means that we can't determine if a LNX has an outdated eth app without the NFT support OR if you're using an LNS that will just not have NFT support (at least for now).